test/e2e-tests.sh

```diff
@@ -189,7 +205,12 @@ kustomize build overlays/dev-locked-down --load_restrictor=LoadRestrictionsNone
 kustomize build overlays/dev-openshift --load_restrictor=LoadRestrictionsNone || fail_test "Failed to run kustomize on overlays/dev-openshift"
 kustomize build overlays/dev-openshift-locked-down --load_restrictor=LoadRestrictionsNone || fail_test "Failed to run kustomize on overlays/dev-openshift-locked-down"
 
-test_overlay dev "proxy"
-test_overlay dev-locked-down "kubectl"
+# test overlays
+test_dashboard overlay proxy dev
+test_dashboard overlay kubectl dev-locked-down
+
+# test installer
+test_dashboard installer proxy
+test_dashboard installer kubectl --readonly
```

Comment:
```suggestion
test_dashboard installer kubectl --read-only
```

Reply:
the actual option is `--readonly` i can change it to `--read-only` if you want

Reply:
ah so it is, yes I think we should change it to be consistent with the dashboard flag

Reply:
changed it

Reply:
still needs to be updated here

Reply:
thanks, good catch.
fixed